src/auth/providers/generateTokensProvider.ts

```
@@ -1,73 +1,61 @@
-import { forwardRef, Inject, Injectable } from '@nestjs/common';
-import { ApiTags, ApiOperation } from '@nestjs/swagger';
+import { Inject, Injectable } from '@nestjs/common';
```

ESLint

[error] 'Inject' is defined but never used.

(@typescript-eslint/no-unused-vars)